tools/SetupFlow/DevHome.SetupFlow/Models/RepositoryProvider.cs

```diff
@@ -108,50 +127,23 @@ public IRepository GetRepositoryFromUri(Uri uri, IDeveloperId developerId = null
     /// Checks with the provider if it understands and can clone a repo via Uri.
     /// </summary>
     /// <param name="uri">The uri to the repository</param>
-    /// <returns>A tuple that containes if the provider can parse the uri and the account it can parse with.</returns>
-    /// <remarks>If the provider can't parse the Uri, this will try a second time with any logged in accounts.  If the repo is
-    /// public, the developerid can be null.</remarks>
-    public (bool, IDeveloperId, IRepositoryProvider) IsUriSupported(Uri uri)
+    /// <returns>True if this provider supports the url.  False otherwise.</returns>
+    public bool IsUriSupported(Uri uri)

```

Comment:
Nit: IsUriSupportedByAnyDeveloperId()
Or maybe clarifying with a comment is enough

Reply:
Added comments explaining the method.